client/app/lib/redux/services/payment.coffee

```diff
@@ -36,6 +36,8 @@ exports.fetchSubscription = fetchSubscription = pickData ->
   client.get Endpoints.SubscriptionGet
 
 exports.createSubscription = createSubscription = pickData (params = {}) ->
+  if not params.trialEnd
+    params.trialEnd = (new Date()).getTime() + (30 * 24 * 60 * 60 * 1000)
```

Comment:
TIL `+(variable)` is how [asm.js](https://github.com/dherman/asm.js/) casts everything into numbers.